<?php
try {

} catch (\BadFunctionCallException $bf) {
} catch (\BadMethodCallException $bm) {}
-----
{main}:
	encounters: []
	try
		encounters: []
	catch#0 catches: [badfunctioncallexception, badmethodcallexception]
	catch#1 catches: []
	uncaught: []
